Calculators/BMI-Calculator/README.md

```diff
@@ -12,4 +12,3 @@ Calculates the Body Mass Index of a person.
 
 ## Screenshots :-
 
```

Comment:
Add the proper screenshot here @VardaPanchal005

Reply:
Work on this @VardaPanchal005

Reply:
is it reflected now?

Reply:
No, push your change and make commit to get it reflected @VardaPanchal005